### PR TITLE
Add --ignore option to the listen command

### PR DESCRIPTION
**Pull request by jgriffiths**

### CODE REVIEW

File: green_cli/common.py [hunk 1]
```diff
@@ -113,14 +113,23 @@ def getsystemmessages(session):
...
 
     Wait indefinitely for notifications from the gdk and print then to the console. ctrl-c to stop.
     """
+    ignore = [s.strip() for s in ignore.split(',')] if ignore else []
+    last = None
     while True:
         try:
-            click.echo(format_output(session.notifications.get(block=True, timeout=1)))
+            n = session.notifications.get(block=True, timeout=1)
+            if n == last or n.get('event', None) in ignore:
```
Comment (jkauffman1) [inline green_cli/common.py:128]: Did you mean to say `if n == last and unique`?

Comment (jgriffiths) [reply green_cli/common.py]: I broke unique when adding ignore, updated now, thanks.

File: green_cli/common.py [hunk 2]
```diff
@@ -113,14 +113,20 @@ def getsystemmessages(session):
 
 @green.command()
 @with_login
-def listen(session):
+@click.option('--ignore', type=str)
```
Comment (jkauffman1) [inline green_cli/common.py:116]:
```suggestion
@click.option('--ignore', type=str, help='comma delimited list of events to ignore, e.g. "block,fees"')
```

File: green_cli/common.py [hunk 3]
```diff
@@ -113,14 +113,20 @@ def getsystemmessages(session):
...
 
     Wait indefinitely for notifications from the gdk and print then to the console. ctrl-c to stop.
     """
+    ignore = [s.strip() for s in ignore.split(',')] if ignore else []
     while True:
         try:
-            click.echo(format_output(session.notifications.get(block=True, timeout=1)))
+            n = session.notifications.get(block=True, timeout=1)
+            if n.get('event', None) in ignore:
+                logging.debug("Ignoring duplicate/filtered notification")
```
Comment (jkauffman1) [inline green_cli/common.py:127]:
```suggestion
                logging.debug("Ignoring filtered notification")
```

**Comment by jkauffman1** [comment]
utack 63dfc1829c9aa4a9b5c1946b9045987ac72e38b2

File: green_cli/common.py (hunk 3 above)
Comment (jgriffiths) [reply green_cli/common.py:127]: updated, thanks

**Pull request merged**